Cargo.toml

```diff
@@ -14,6 +14,7 @@ libc = "0.2"
 sct = "0.7"
 rustls-pemfile = "0.2.1"
 log = "0.4.14"
+num_enum = "0.5.4"
```

Comment:
IMO the extra dependency for this is overkill. In this case it seems fine to write a custom `TryFrom` impl based on the repr type.